src/boot/fault.c

```diff
@@ -548,7 +546,7 @@ void Fault_Wait5Seconds(void) {
 
     do {
         Fault_Sleep(1000 / 60);
-    } while ((osGetTime() - start) <= OS_SEC_TO_CYCLES(5));
+    } while ((osGetTime() - start) <= OS_USEC_TO_CYCLES(5 * 1000 * 1000));
```

Comment:
Why this changed?

Reply:
I removed `OS_SEC_TO_CYCLES` since it is not from original libultra, while `OS_USEC_TO_CYCLES` is.